code/parse/sexp.h

```diff
@@ -1423,6 +1423,7 @@ extern bool sexp_recoverable_error(int num);
 extern const char *sexp_error_message(int num);
 extern int count_free_sexp_nodes();
 
+
```

Comment:
Optional: maybe remove if not needed.

Reply:
The blank line is intentional - to set off the prototypes in different groups